samples-aspnetcore-6x/resource-server/README.md

```diff
@@ -0,0 +1,64 @@
+* One of our front-end sample applications to demonstrate the interaction with the resource server:
+  * [Okta Angular Sample Apps][]
+  * [Okta React Sample Apps][]
+  * [Okta Vue Sample Apps][]
+
+*Note:* This sample is pre-configured with an open CORS policy to make it easy to test with frontend projects. Make sure to restrict it to known origins in your production applications.
+
+A typical resource-server requires a frontend and a backend application, so you will need to start each process as described in the next section.
+
+## Running This Example
+
+### Backend
+
+Run the example with your preferred tool and write down the port of your Web API application to configure Okta afterwards.
+
+> **NOTE:** Although we highly recommend using HTTPS in production, we are using HTTP here to be compatible with our [SPA samples](#Prerequisites) that expect a protected endpoint on HTTP://localhost:7064/api/whoami. 
+
+#### Run the Web API application from Visual Studio
+
+If you run this project in Visual Studio it will start the resource server on port 7064 using HTTP. You can change this configuration in the `launchSettings.json` in the Properties folder.  
+You can browse to `http://localhost:7064/api/hello` to ensure it has started. You will see the message "You are anonymous". When you browse to `http://localhost:7064/api/whoami` you will get a 401 HTTP response.  An access token is needed to access the protected resource which is acquired by the front-end, see [Front-end](#front-end).
+
+#### Run the Web API application from dotnet CLI
+
+If you run this project via the dotnet CLI it will start the resource server on port 7064 using HTTP. You can change this configuration in the `launchSettings.json` in the Properties folder.
+
+Navigate to the folder where the project file is located and type the following:
+
+```dotnet run```
+
+You can browse to `http://localhost:7064/api/hello` to ensure it has started. You will see the message "You are anonymous". When you browse to `http://localhost:7064/api/whoami` you will get a 401 HTTP response.  An access token is needed to access the protected resource which is acquired by the front-end, , see [Front-end](#front-end).
+
+### Add your Okta configuration to the sample's appsettings
+
+Replace the okta configuration placeholders in the `appsettings.json` with your configuration values from the [Okta Developer Console]. 
+You can see all the available configuration options in the [okta-aspnet GitHub](https://github.com/okta/okta-aspnet/blob/master/docs/aspnetcore-webapi.md#configuration-reference).
+
+### Front-end
+
+If you want to use one of our front-end samples, open a new terminal window and run the [front-end sample project of your choice](#Prerequisites).  
+Make sure to update the resource server URI configuration to use your configured resource-server port (for example `http://localhost:8000`). 
```

Comment:
So, since we recommend ppl to use our frontend apps to test the resource server sample, we have to add the messages endpoint (similarly to what we had in existing samples). Those samples assume you have a resource server with a message endpoint.